src/material-experimental/mdc-input/_input-theme.scss

```diff
@@ -1,7 +1,25 @@
+@import '../../material/core/theming/theming';
 @import '../mdc-helpers/mdc-helpers';
 
 @mixin mat-mdc-input-theme($theme) {
-  @include mat-using-mdc-theme($theme) {}
+  @include mat-using-mdc-theme($theme) {
+    .mat-mdc-input-element {
+      // On dark themes we set the native `select` color to some shade of white,
```

Comment:
Hmm, I wonder if this will still be an issue if you have your browser in dark mode, or maybe it does some magic color inverting stuff for you - anyways if it is an issue it seems like a bigger thing than just this case

Reply:
Just FYI: On macOS with the OS configured for dark mode (and Material using a dark mode), these styles are completely ignored for the native select dropdowns.

![Screen Shot 2021-06-19 at 01 46 19](https://user-images.githubusercontent.com/3506071/122632354-525b6c00-d0a0-11eb-92b5-c9ddecce95dc.png)

Even if I try to override these styles to `red`, it does nothing in the native select dropdown.